src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -11,6 +11,11 @@ import {
   withSavedState,
 } from "/core/utils.js";
 
+// the following icon SVG path code is from https://tabler.io/icons
```

Comment:
Please move this section to just before the `registerVisualizationLayerDefinition` where you use it.

Reply:
done.